backend/fms_core/serializers.py

```diff
@@ -275,11 +275,9 @@ class Meta:
         depth = 1
 
 class ProjectSerializer(serializers.ModelSerializer):
-    samples = serializers.PrimaryKeyRelatedField(read_only=True, many=True)
     class Meta:
         model = Project
-        fields = "__all__"
-        extra_fields = ('samples')
+        fields = ("id", "name", "principal_investigator", "requestor_name", "requestor_email", "targeted_end_date", "status")
 
 class ProjectExportSerializer(serializers.ModelSerializer):
     samples = serializers.PrimaryKeyRelatedField(read_only=True, many=True)
```

Comment:
We should remove or fix the samples in the export. It is not working at the moment... Not sure we need the list of samples here.